app/dashboard/models.py

```diff
@@ -2105,6 +2129,12 @@ class Profile(SuperModel):
         help_text=_('override profile avatar'),
     )
     dominant_persona = models.CharField(max_length=25, choices=PERSONAS, blank=True)
+    longest_streak = models.IntegerField(default=0)
+    activity_level = models.CharField(max_length=10, blank=True, help_text=_('the users activity level (high, low, new)'))
```

Comment:
did you forget `medium`? (I guess it's only the help text)

Reply:
in the help text only yes